### PR TITLE
Update external-data.mdx with Why

### PR DESCRIPTION
**Pull request by cmo-pomerium**
Added section explaining Why, link to docs

### CODE REVIEW

File: content/docs/enterprise/external-data.mdx [hunk 1]
```diff
@@ -4,6 +4,11 @@ sidebar_label: External Data
 description: Extend your authorization policies with data from external sources.
 ---
 
+If you believe decisions should be informed by data, then security decisions should be informed by **all** the data. Today, most applications are limited to using user identity when making access decisions — which leaves systems blind to the multi-faceted forms of attack vectors being exploited in the threat landscape. It turns out an access control system is only as good as the data used in policy decisions.
+
+For more information on how context enriches access decisions, read our [blog post here.](https://www.pomerium.com/blog/context-drives-full-access-decision-making/)
```
Comment (alexfornuto) [inline content/docs/enterprise/external-data.mdx:9]:
```suggestion
For more information on how context enriches access decisions, read our [blog post here](https://www.pomerium.com/blog/context-drives-full-access-decision-making/).
```

File: content/docs/enterprise/external-data.mdx [hunk 2]
```diff
@@ -4,6 +4,11 @@ sidebar_label: External Data
...
+
+
```
Comment (alexfornuto) [inline content/docs/enterprise/external-data.mdx:11]: Extraneous newline

Comment (cmo-pomerium) [inline content/docs/enterprise/external-data.mdx:11]:
```suggestion
```

**Review by cmo-pomerium — COMMENTED** [review]
Resolved changes.

File: content/docs/enterprise/external-data.mdx (hunk 2 above)
Comment (alexfornuto) [reply content/docs/enterprise/external-data.mdx:11]: @cmo-pomerium it's your own change, go ahead and commit it.

**Review by alexfornuto — COMMENTED** [review]
LGTM, but tagging in @desimone to confirm that he's on board with this sort of intro to docs.

**Review by desimone — APPROVED** [review]

**Pull request merged**